src/ipc/ipc4/helper.c

```diff
@@ -793,6 +831,7 @@ int ipc4_trigger_chain_dma(struct ipc *ipc, struct ipc4_chain_dma *cdma)
 	return IPC4_SUCCESS;
 }
 
+#include "mem_win_debug.h"
```

Comment:
it looks like a of of debug logic has been left in this PR and could be removed ?

Reply:
Reolved in:
![image](https://user-images.githubusercontent.com/93590538/203565472-5812994b-5142-4407-a1c6-744b2d414e71.png)